src/sentry/tasks/relay.py

```diff
@@ -251,6 +251,7 @@ def schedule_invalidate_project_config(
     project_id=None,
     public_key=None,
     countdown=5,
+    transaction_db="",
```

Comment:
I think it has to be `None` when we pass it into `on_commit`, because that function in turn calls `get_connection`, which expects `None` if I'm not mistaken:

https://github.com/django/django/blob/9cc0d7f7f85cecc3ad15bbc471fe6a08e4f515b6/django/db/transaction.py#L23-L24

Reply:
Passing `None` to `on_commit` will result in an exception being thrown since [we've patched](https://github.com/getsentry/sentry/blob/8deb98a50929293c9973e6da39b31164d851e67a/src/sentry/silo/patches/silo_aware_transaction_patch.py#L54) `transaction.get_connection` to expect an explicit `using` parameter now. This is why we fallback to using the DB backing the `Project` model instead as a safe default.